phantom_connector.py

```diff
@@ -953,13 +965,13 @@ def _deflate_item(self, param):
                                 "Failed to get vault item info: {}".format(self._get_error_message_from_exception(e)))
 
         try:
-            file_type = magic.from_file(file_path, mime=True)
+            file_type, is_supported = self.is_deflation_supported_file(file_path)
```

Comment:
When I see `is_...` I expect the function to return a `bool`, not a `tuple`

Reply:
Good note